src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/fieldTransformers/dateTimeFieldTransformer.scss

```diff
@@ -0,0 +1,10 @@
+@import '../../Application/colors.scss';
+
+.date-time {
```

Comment:
i think there wasnt any css before - is this the same format as used before?

Reply:
yes but removed it